sql/service/src/main/scala/org/apache/spark/sql/service/SparkThriftServer2.scala

```diff
@@ -289,34 +286,15 @@ object SparkThriftServer2 extends Logging {
 }
 
 private[spark] class SparkThriftServer2(sqlContext: SQLContext)
-  extends SparkServer2(sqlContext)
-  with ReflectedCompositeService {
```

Comment:
Could we remove `ReflectedCompositeService`?

Reply:
> Could we remove `ReflectedCompositeService`?

Has been removed in SparkSQLCLIService